addon/templates/components/cfb-form-editor/question.hbs

```diff
@@ -0,0 +1,71 @@
+{{#if data.isRunning}}
+  <div class="uk-flex uk-flex-center uk-flex-middle uk-height-small">
+    {{uk-spinner ratio=2}}
+  </div>
+{{else}}
+  {{#with data.lastSuccessful.value as |model|}}
+    {{#validated-form
+      model=(changeset model validations)
+      on-submit=(perform submit)
+    as |f|}}
+      <div class="uk-alert-warning" uk-alert>
+        <a class="uk-alert-close" uk-close></a>
+        <div uk-grid class="uk-grid-small">
+          <div class="uk-width-auto uk-flex uk-flex-middle">
+          {{uk-icon 'warning' ratio=1.2}}
+          </div>
+          <div class="uk-width-expand">
+            <strong>{{t 'caluma.form-builder.question.minor-info-title'}}</strong>
+            {{t 'caluma.form-builder.question.minor-info'}}
+          </div>
+        </div>
+      </div>
+
+      {{#f.input name='label' as |i|}}
+        <label class="uk-form-label uk-text-bold">{{t 'caluma.form-builder.question.label'}}</label>
+        <input
+          name={{i.name}}
+          class="uk-input {{i.class}}"
+          value={{i.value}}
+          onblur={{i.setDirty}}
+          oninput={{action i.update value='target.value'}}
+        >
```

Comment:
Why do the inputs need to be custom?

Reply:
UIkit has a different class for every type of input element. `ember-validated-form` does not support this.